src/main/java/org/gridsuite/shortcircuit/server/entities/AnalysisParametersEntity.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.shortcircuit.server.entities;
+
+import com.powsybl.shortcircuit.InitialVoltageProfileMode;
+import com.powsybl.shortcircuit.StudyType;
+import jakarta.persistence.*;
+import lombok.*;
+import lombok.experimental.Accessors;
+import org.gridsuite.shortcircuit.server.dto.ShortCircuitPredefinedConfiguration;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Objects;
+import java.util.UUID;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+
+/**
+ * @since 1.7.0
+ */
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Accessors(chain = true)
+@Getter
+@Setter
+@Entity
+@Table(name = "shortcircuit_parameters")
+public class AnalysisParametersEntity {
```

Comment:
Not convinced by the interest of putting default values in the DB directly maybe we should raise this point in the technical meeting

Reply:
Can you rename ShortCircuitParametersEntity ?